definitions/v4/miatrix.yml

```diff
@@ -0,0 +1,169 @@
+    - {id: 2050, cat: Movies/3D, descr: Movies/3D}
+    - {id: 2060, cat: Movies/BluRay, descr: Movies/BluRay}
+    - {id: 2070, cat: Movies/DVD, descr: Movies/DVD}
+    - {id: 2010, cat: Movies/Foreign, descr: Movies/Foreign}
+    - {id: 2040, cat: Movies/HD, descr: Movies/HD}
+    - {id: 2020, cat: Movies/Other, descr: Movies/Other}
+    - {id: 2030, cat: Movies/SD, descr: Movies/SD}
+    - {id: 2045, cat: Movies/UHD, descr: Movies/UHD}
+    - {id: 3000, cat: Audio, descr: Audio}
+    - {id: 3030, cat: Audio/Audiobook, descr: Audio/Audiobook}
+    - {id: 3060, cat: Audio/Foreign, descr: Audio/Foreign}
+    - {id: 3040, cat: Audio/Lossless, descr: Audio/Lossless}
+    - {id: 3010, cat: Audio/MP3, descr: Audio/MP3}
+    - {id: 3050, cat: Audio/Other, descr: Audio/Other}
+    - {id: 3020, cat: Audio/Video, descr: Audio/Video}
+    - {id: 4000, cat: PC, descr: PC}
+    - {id: 4010, cat: PC/0day, descr: PC/0day}
+    - {id: 4050, cat: PC/Games, descr: PC/Games}
+    - {id: 4020, cat: PC/ISO, descr: PC/ISO}
+    - {id: 4030, cat: PC/Mac, descr: PC/Mac}
+    - {id: 4070, cat: PC/Mobile-Android, descr: PC/Mobile-Android}
+    - {id: 4040, cat: PC/Mobile-Other, descr: PC/Mobile-Other}
+    - {id: 4060, cat: PC/Mobile-iOS, descr: PC/Mobile-iOS}
+    - {id: 5000, cat: TV, descr: TV}
+    - {id: 5070, cat: TV/Anime, descr: TV/Anime}
+    - {id: 5080, cat: TV/Documentary, descr: TV/Documentary}
+    - {id: 5020, cat: TV/Foreign, descr: TV/Foreign}
+    - {id: 5040, cat: TV/HD, descr: TV/HD}
+    - {id: 5050, cat: TV/Other, descr: TV/Other}
+    - {id: 5030, cat: TV/SD, descr: TV/SD}
+    - {id: 5060, cat: TV/Sport, descr: TV/Sport}
+    - {id: 5045, cat: TV/UHD, descr: TV/UHD}
+    - {id: 5010, cat: TV/WEB-DL, descr: TV/WEB-DL}
+    - {id: 6000, cat: XXX, descr: XXX}
+    - {id: 6010, cat: XXX/DVD, descr: XXX/DVD}
+    - {id: 6060, cat: XXX/ImageSet, descr: XXX/ImgSet}
+    - {id: 6070, cat: XXX/Other, descr: XXX/Other}
+    - {id: 6050, cat: XXX/Pack, descr: XXX/Pack}
+    - {id: 6020, cat: XXX/WMV, descr: XXX/WMV}
+    - {id: 6030, cat: XXX/XviD, descr: XXX/XviD}
+    - {id: 6040, cat: XXX/x264, descr: XXX/x264}
+    - {id: 7000, cat: Books, descr: Books}
+    - {id: 7030, cat: Books/Comics, descr: Books/Comics}
+    - {id: 7020, cat: Books/EBook, descr: Books/Ebook}
+    - {id: 7060, cat: Books/Foreign, descr: Books/Foreign}
+    - {id: 7010, cat: Books/Mags, descr: Books/Mags}
+    - {id: 7050, cat: Books/Other, descr: Books/Other}
+    - {id: 7040, cat: Books/Technical, descr: Books/Technical}
+    - {id: 8000, cat: Other, descr: Other}
+    - {id: 8020, cat: Other/Hashed, descr: Other/Hashed}
+    - {id: 8010, cat: Other/Misc, descr: Other/Misc}
+
+  modes:
+    # Be sure to update the search inputs
+    search: [q] # group is supported by the indexer but not Cardigann/Prowlarr
+    tv-search: [q, rid, tvdbid, tvmazeid, season, ep]
+    movie-search: [q, imdbid] # genre is supported by the indexer but not Cardigann/Prowlarr
```

Comment:
We can likely add these extra features pretty easily on our side as part of the PR on the other repo.